### PR TITLE
PWGGA/GammaConv Add timerange cut to AliConvEventCuts

### PR DESCRIPTION
**Pull request by sstiefel19**
tested with /alice/data/2018/LHC18r/000297481/pass3/AOD/001 (time
cut cuts some events) and /alice/data/2018/LHC18r/000297414/pass3/AOD/001
(no events being cut)

### CODE REVIEW

File: PWGGA/GammaConvBase/AliConvEventCuts.h [hunk 1]
```diff
@@ -671,6 +669,7 @@ class AliConvEventCuts : public AliAnalysisCuts {
       TString*                    fGeneratorNames;                        //[fnHeaders]
       PeriodVar                   fPeriodEnum;                            ///< period selector
       EnergyVar                   fEnergyEnum;                            ///< energy selector
+      AliTimeRangeCut             fTimeRangeCut;
```
Comment (dmuhlhei) [inline PWGGA/GammaConvBase/AliConvEventCuts.h:672]: please use the streamer //! or update the classdef

File: PWGGA/GammaConvBase/AliConvEventCuts.cxx [hunk 2]
```diff
@@ -117,6 +117,7 @@ AliConvEventCuts::AliConvEventCuts(const char *name,const char *title) :
   fGeneratorNames(NULL),
   fPeriodEnum(kNoPeriod),
   fEnergyEnum(kUnset),
+  fTimeRangeCut(),
```
Comment (dmuhlhei) [inline PWGGA/GammaConvBase/AliConvEventCuts.cxx:120]: can you please check if this throws a warning during compilation?

Comment (sstiefel19) [reply PWGGA/GammaConvBase/AliConvEventCuts.cxx:120]: checked, does not throw a warning

File: PWGGA/GammaConvBase/AliConvEventCuts.h (hunk 1 above)
Comment (sstiefel19) [reply PWGGA/GammaConvBase/AliConvEventCuts.h:672]: okay, forgot the clasdef again. What is the streamer //! ? How would that work?

**Comment by dmuhlhei** [comment]
+1

**Pull request merged**